packages/create-app/templates/default-app/packages/app-next/README.md

```diff
@@ -0,0 +1,9 @@
+# example-app
+
+This package is an example of a Backstage application using
+the [new frontend](https://backstage.io/docs/frontend-system/).
+
+To play with it, open a terminal and run the command: `yarn start`
+
+**NOTE:** Don't forget to open a second terminal and to launch the backend there, using `yarn start`! The frontend
```

Comment:
Would it make sense to include how to add a script to be able to run `yarn dev-next` like we do here: https://github.com/backstage/community-plugins/blob/984800cdb71c7eda8554270f3a868c425ed538ba/workspaces/github-actions/package.json#L10

Reply:
Now that I've worked with this my comment doesn't apply: this flag creates a new backstage instance with out the `app` and create an `app-next` using the new frontend system along with the current backend. 👍